package.json

```
@@ -37,7 +37,7 @@
     "grunt-retire": "^1.0.7",
     "grunt-zip": "~0.17.1",
     "mustache": "^2.3.0",
-    "socket.io": "^1.7.3"
+    "socket.io": ">=2.4.0"
```

Oops, I think the syntax here would be
```suggestion
    "socket.io": "^2.4.0"
```